spaces/domains.json

```
@@ -371,4 +371,5 @@
   "vote.meowfinance.org": "meowfinance.eth",
   "gov.dot.finance": "dotfinance.eth",
   "vote.mgh.dao": "metagamehub.eth"
+  "gov.maidcoin.org": "maidcoin.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "vote.mgh.dao": "metagamehub.eth",
  "gov.maidcoin.org": "maidcoin.eth"
```